/root/miniconda3/envs/scv/lib/python3.9/site-packages/torch/functional.py:445: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  ../aten/src/ATen/native/TensorShape.cpp:2157.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Setting up Multi Scale Gradient loss...
Done
---- Single Training ----
Use GPU: 0 for training
Found 26 samples in /root/autodl-tmp/Spike-Stero/train
Found 8 samples in /root/autodl-tmp/Spike-Stero/validation
-----  [3, 6, 12]
---- new version 4 ----
Model Initialized
Loading initial model weights from: /root/autodl-tmp/SpikeT/train_s2d_SpikeTransformer/model_best.pth.tar
Traceback (most recent call last):
  File "/root/code/smde/train.py", line 333, in <module>
    main()
  File "/root/code/smde/train.py", line 328, in main
    main_worker(args.gpu, ngpus_per_node, args)
  File "/root/code/smde/train.py", line 252, in main_worker
    checkpoint = torch.load(initial_checkpoint)
  File "/root/miniconda3/envs/scv/lib/python3.9/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/root/miniconda3/envs/scv/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/root/miniconda3/envs/scv/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/root/autodl-tmp/SpikeT/train_s2d_SpikeTransformer/model_best.pth.tar'
